src/server/areImagesEqual.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = function areImagesEqual(a, b) {
+  if (a.height !== b.height) {
+    return false;
+  }
+  if (a.width !== b.width) {
+    return false;
+  }
+  const len = a.data.length;
+  for (let i = 0; i < len; i += 1) {
```

Comment:
Although, I wonder if we can figure out how to take screenshots of things without a page background color so that there is transparency. That would make it possible to differentiate between a white background that a component adds and transparent background on the component.